lib/puppet/provider/posix_acl/posixacl.rb

```diff
@@ -77,7 +77,7 @@ def check_set
     @resource.value(:action) == :set
   end
 
-  def permission=(value)
+  def permission=(_value) # TODO: Investigate why we're not using this parameter
```

Comment:
This looked a bit fishy.  I think rubocop is correct that `value` was not being used, but perhaps it *should* be?  Dunno, hence the comment.